src/vm/docs/mod.rs

```diff
@@ -393,6 +393,14 @@ principal, and executes `expr` with that context. It returns the resulting value
     example: "(as-contract (print tx-sender)) => 'CTcontract.name"
 };
 
+const GET_BLOCK_INFO_API: SpecialAPI = SpecialAPI {
```

Comment:
These need to be filled in before this PR is accepted.

Reply:
Filled in the docs. Added a note/warning about block time usage.